src/scales/scale.time.js

```diff
@@ -501,7 +508,7 @@ module.exports = Scale.extend({
 		if (raw === undefined) {
 			return NaN;
 		}
-		return toTimestamp(this, raw);
+		return parse(this, raw);
```

Comment:
`toTimestamp` is now called only from `parse`, so I would inline it

Reply:
done